make/common/TestFilesCompilation.gmk

```diff
@@ -103,10 +107,21 @@ define SetupTestFilesCompilationBody
         OPTIMIZATION := $$(if $$($1_OPTIMIZATION_$$(name)),$$($1_OPTIMIZATION_$$(name)),LOW), \
         COPY_DEBUG_SYMBOLS := false, \
         STRIP_SYMBOLS := $$(if $$($1_STRIP_SYMBOLS_$$(name)),$$($1_STRIP_SYMBOLS_$$(name)),false), \
+        BUILD_INFO_LOG_MACRO := LogInfo, \
     )) \
-    $$(eval $1 += $$(BUILD_TEST_$$(name)) )  \
+    $$(eval $1 += $$(BUILD_TEST_$$(name)) ) \
+    $$(eval $1_BUILD_INFO_DEPS += $$(BUILD_TEST_$$(name)_BUILD_INFO_DEPS)) \
+    $$(eval $$(BUILD_TEST_$$(name)_BUILD_INFO) :| $$($1_BUILD_INFO)) \
```

Comment:
```suggestion
    $$(eval $$(BUILD_TEST_$$(name)_BUILD_INFO): | $$($1_BUILD_INFO)) \
```

The formatting threw me off here, wondering what kind of operator `:|` were. We usually have no space between target name and `:`, and a space before `|` for order-only, so I recommend to keep that style here.